Octokit.Tests.Integration/Clients/ReleasesClientTests.cs

```diff
@@ -372,6 +639,21 @@ public TheUploadAssetMethod()
         }
 
         [IntegrationTest]
+        public async Task ReturnsCorrectCountOfReleasesWithStartWithRepositoryId()
+        {
+            var options = new ApiOptions
+            {
+                PageSize = 1,
+                PageCount = 1,
+                StartPage = 1
```

Comment:
Ah here it is!  This is set to `StartPage = 1` here, but in the non repository Id test it is `2`.  It would be good to have these consistent (eventhough it doesnt really affect the result)